private/buf/bufprint/bufprint.go

```diff
@@ -73,66 +78,180 @@ func (f Format) String() string {
 	}
 }
 
-// CuratedPluginPrinter is a printer for curated plugins.
-type CuratedPluginPrinter interface {
-	PrintCuratedPlugin(ctx context.Context, format Format, plugin *registryv1alpha1.CuratedPlugin) error
-	PrintCuratedPlugins(ctx context.Context, format Format, nextPageToken string, plugins ...*registryv1alpha1.CuratedPlugin) error
+// OutputEntity is an entity printed structurally by functions in bufprint package.
+// It's used in "buf registry" commands where the CLI prints a page of entities, such as
+// commits, an entity's info or simply an entity's full name.
+//
+// An implementation of OutputEntity must also be a struct. If a field should be printed
+// in table form, add a field tag with bufprint:"<field name>[,omitempty]".
+type OutputEntity interface {
+	fullName() string
 }
 
-// NewCuratedPluginPrinter returns a new CuratedPluginPrinter.
-func NewCuratedPluginPrinter(writer io.Writer) CuratedPluginPrinter {
-	return newCuratedPluginPrinter(writer)
+// Print prints entities' names.
```

Comment:
`PrintNames`?

Reply:
resolved in [785f89b](https://github.com/bufbuild/buf/pull/3173/commits/785f89b564deb9d4752363a922a70c7bdebdb572)